From 66979a040e41c7890435c084f9ff12f7806e9dd2 Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Thu, 12 Feb 2009 16:02:17 +0000
Subject: [PATCH] Remove long-gone '-co' option from Xserver man page

The '-co' option was removed in commit dda10c9066a660b647384179f82e1da8e063264f

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
(cherry picked from commit b0ad9e1ced9619f37acf77764c395c57b86cf463)
---
 doc/Xserver.man.pre |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/doc/Xserver.man.pre b/doc/Xserver.man.pre
index 8fe3b76..5861736 100644
--- a/doc/Xserver.man.pre
+++ b/doc/Xserver.man.pre
@@ -117,10 +117,6 @@ sets key-click volume (allowable range: 0-100).
 sets the visual class for the root window of color screens.
 The class numbers are as specified in the X protocol.
 Not obeyed by all servers.
-.TP 8
-.B \-co \fIfilename\fP
-sets name of RGB color database.  The default is
-.IR __projectroot__/share/X11/rgb .
 .ig
 .TP 8
 .B \-config \fIfilename\fP
-- 
1.5.6.5

From 202b7e9644ce79955f24fb610a5da2dac1e51998 Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Thu, 12 Feb 2009 16:03:16 +0000
Subject: [PATCH] Remove references to rgb.txt from files section of Xserver and Xorg man pages

The references to this file in the server code were removed in commit dda10c9066a660b647384179f82e1da8e063264f

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
(cherry picked from commit d0dd649035fc3698c5b436f9d9d248116aa106a3)
---
 doc/Xserver.man.pre             |    3 ---
 hw/xfree86/doc/man/Xorg.man.pre |    3 ---
 2 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/doc/Xserver.man.pre b/doc/Xserver.man.pre
index 5861736..5d1f7b0 100644
--- a/doc/Xserver.man.pre
+++ b/doc/Xserver.man.pre
@@ -553,9 +553,6 @@ Bitmap font directories
 .IR __projectroot__/lib/X11/fonts/TTF , __projectroot__/lib/X11/fonts/Type1
 Outline font directories
 .TP 30
-.I __projectroot__/share/X11/rgb.txt
-Color database
-.TP 30
 .I /tmp/.X11-unix/X\fBn\fP
 Unix domain socket for display number \fBn\fP
 .TP 30
diff --git a/hw/xfree86/doc/man/Xorg.man.pre b/hw/xfree86/doc/man/Xorg.man.pre
index 504552c..b96dc0a 100644
--- a/hw/xfree86/doc/man/Xorg.man.pre
+++ b/hw/xfree86/doc/man/Xorg.man.pre
@@ -560,9 +560,6 @@ Libraries.
 .B __projectroot__/lib/X11/fonts/\(**
 Fonts.
 .TP 30
-.B __projectroot__/share/X11/rgb.txt
-Color names to RGB mapping.
-.TP 30
 .B __projectroot__/share/X11/XErrorDB
 Client error message database.
 .TP 30
-- 
1.5.6.5

